bin/pycbc_multi_inspiral

```diff
@@ -330,6 +338,16 @@ init_logging(args.verbose)
 num_slides = slide_limiter(args)
 # Arrange detectors alphabetically so they are always called in the same order
 args.instruments.sort()
+# Check that the number of IFOs requested to have single SNR above threshold
+# is compativle with the number of detectors in the analysis
+if len(args.instruments) > 1:
```

Comment:
I think the logic is still correct even without this check. If you only have one detector and you ask `--nifo-sngl-snr-threshold 2`, raising the error is still the correct behavior, no?

Reply:
I added this `if` because later on if `nifo` (which is equal to `len(args.instruments)`) is 1 the single IFO SNR cut is not used, so  `--nifo-sngl-snr-threshold` can be anything in that case and the code will still work (correctly).

Reply:
I see the point, though I think this makes the CLI confusing. Perhaps this is something else to explain in the help for `--nifo-sngl-snr-threshold`.

One more thing, there is no need to nest the two if statements, and there is an extra call to `parser.error()`.